python/paddle/v2/framework/tests/test_add_two_op.py

```diff
@@ -13,8 +13,8 @@ class TestAddOp(unittest.TestCase):
     def setUp(self):
         self.type = "add_two"
         self.inputs = {
-            'X': numpy.random.random((102, 105)).astype("float32"),
-            'Y': numpy.random.random((102, 105)).astype("float32")
+            'X': numpy.random.random((12, 15)).astype("float32"),
```

Comment:
Please add cases like 

- [7, 0]
- [0, 7]
- [0, 0]
- [7, 13]  

7  and 13 are just small prime numbers.